content/blog/2025/02/2025-02-20-crossroads.md

```diff
@@ -0,0 +1,115 @@
+
+* Without Trust & Safety efforts, bad actors and communities would proliferate on the network and make it unlivable for the rest.
+* Without a canonical specification, the shared infrastructure and a Spec Core Team to maintain it, the protocol would become fragmented, losing its effective interoperability – increasing the costs on all downstream users.
+* Without a neutral entity as the custodian of the specification, the ecosystem would first shatter and then consolidate around the biggest (likely for-profit) actor.
+* Without advocacy, conferences, documentation and tutorials, Matrix would become a niche protocol used by a few enthusiasts for side projects, whilst big proprietary and siloed networks continue to hold the world’s communications.
+
+
+## Implementing the vision
+
+But there is light at the end of the tunnel! Concretely, the Foundation delivers most of its value by fostering a healthy, fair and fertile ecosystem around Matrix. It needs to strike the right balance between:
+
+* **Making Matrix accessible & visible.**
+    * For the general public it means maintaining an easy default onboarding server (Matrix.org).
+    * For server administrators it means providing the right tooling to keep their users (and themselves!) safe.
+    * For developers it means making it easy to develop products using Matrix, via documentation, tutorials, and in-person events.
+* **Making Matrix compelling to build on.**
+    * This means maintaining the Matrix Specification as a canonical, unencumbered, patent free and royalty free specification.
+    * Being responsive and vendor-neutral when an organisation or individual contributes.
+    * Promoting the good players within the ecosystem.
+    * Ensuring the network grows and attracts more users.
+* **Making Matrix a product that benefits the greater good.**
+    * This means ensuring that the general public can easily build safe & easy to use communities on Matrix.
+    * Ensuring that bad actors are proactively chased and discouraged to use Matrix.
+
+### Doing less to do better
+
+Matrix has been here for 10 years, and will hopefully be here for many more! But to continue to grow and thrive, it needs the Foundation to be around and healthy, which means carefully allocating its budget in order to continue to exist and fulfill its mission. This is why it needs to focus on critical programs and shut down some of its activities.
+
+We view the following programs as critical to the Foundation’s mission:
+
+* Maintaining the canonical, backwards compatible, stable [Matrix Spec](https://spec.matrix.org/latest/)
+* Developing protocol enhancements and Trust and Safety tooling, making the tools available to the ecosystem and moderating the servers under its control (typically Matrix.org) - [see our recent blog post](/blog/2025/02/building-a-safer-matrix/)
+* Running the Matrix.org homeserver as an initial home for newbies
```

Comment:
this might be intended to sound cute, but i think it could be misunderstood
```suggestion
* Running the Matrix.org homeserver as an initial home for newcomers
```